paasta_tools/contrib/get_running_task_allocation.py

```diff
@@ -151,14 +155,18 @@ def get_kubernetes_metadata(
     Optional[str],
     Optional[str],
     Optional[str],
+    Optional[str],
+    Optional[str],
 ]:
     labels = pod.metadata.labels or {}
     pod_name = pod.metadata.name
     pod_ip = pod.status.pod_ip
     host_ip = pod.status.host_ip
     service = labels.get("paasta.yelp.com/service")
     instance = labels.get("paasta.yelp.com/instance")
-    return service, instance, pod_name, pod_ip, host_ip
+    git_sha = labels.get("paasta.yelp.com/git_sha")
+    config_sha = labels.get("paasta.yelp.com/config_sha")
```

Comment:
Quick note (and not sure if this takes care of that) but 'paasta.yelp.com/config_sha' is not set on all pods, specifically, it is not set on pods of cassandra_k8s.  (git_sha is always set though)


Would be good to check how config_sha is checked in rest of paasta code which takes care of optionality of this.

Reply:
@ronin13 I think we are ok with situation when git_sha and/or git_sha are not set. The idea behind this change is to increase granularity for task allocation data for services during restarts and we basically care about ones with git_sha set. If git_sha and/or git_sha labels do not exist we are just at a status quo.